marlowe-protocols/marlowe-protocols.cabal

```diff
@@ -73,3 +77,39 @@ library
     , template-haskell
     , text
     , typed-protocols
+
+test-suite marlowe-protocols-test
+  import: lang
+  hs-source-dirs:
+    test
+  type: exitcode-stdio-1.0
+  main-is: Spec.hs
+  other-modules:
+    Network.Protocol.ChainSeek.CodecSpec
+    Network.Protocol.Job.CodecSpec
+    Network.Protocol.Query.CodecSpec
+    Paths_marlowe_protocols
+  build-depends:
+      base >= 4.9 && < 5
+    , async
+    , binary
+    , bytestring
+    , cardano-api
+    , containers
+    , hspec
+    , marlowe
+    , marlowe-protocols
+    , plutus-tx
+    , plutus-ledger-api
+    , QuickCheck
+    , some
+    , stm
+    , tasty
+    , tasty-hunit
+    , tasty-quickcheck
```

Comment:
We don't need to depend on `tasty` if we're using `hspec`

Reply:
I added `-Wunused-packages` and fixed deps.